tests/nested/manual/core20-fde-dbx/task.yaml

```diff
@@ -0,0 +1,148 @@
+summary: Verify EFI DBX updates in a Core20+ system using FDE
+
+details: |
+    Check that the EFI DBX can be updated in a system using FDE
+
+systems: [ubuntu-2*]
+
+environment:
+    NESTED_ENABLE_TPM: true
+    NESTED_ENABLE_SECURE_BOOT: true
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+    NESTED_UBUNTU_SEED_SIZE: 1500M
+
+prepare: |
+    # TODO copy nested vars file to "$NESTED_ASSETS_DIR/OVMF_VARS.snakeoil.fd"
+    #
+    tests.nested build-image core
+    tests.nested create-vm core    
+
+execute: |
+    echo "Establish initial state"
+    remote.exec sudo cat /var/lib/snapd/device/fde/boot-chains > boot-chains-before.json
+    reseal_count_start="$(jq -r '.["reseal-count"]' < boot-chains-before.json )"
+
+    fetch_and_check_reseal_count_equal() {
+        local reseal_count_now
+        remote.exec sudo cat /var/lib/snapd/device/fde/boot-chains > boot-chains.json
```

Comment:
I wonder if we should keep some thing in the fde state that "snap debug" can access to instead of using this. We should keep this file eventually.